NetflixRouletteWebpack/src/components/results-filter/ResultsFilter.js

```diff
@@ -1,40 +1,94 @@
-import React, { useState } from "react";
+import React from "react";
 import PropTypes from "prop-types";
 import "./ResultsFilter.scss";
+import { connect } from "react-redux";
+import {
+  getSelectedFilters,
+  getSortingOrder,
+  getSortingType,
+} from "../../store/selectors";
+import {
+  setSelectedFilters,
+  sortAndFilterResults,
+} from "../../store/actionCreators";
 
-const DEFAULT_VALUE = "All";
-function ResultsFilter({ genresFilterArray }) {
-  const [selectedFilter, setSelectedFilter] = useState(DEFAULT_VALUE);
-
-  const handleGenreFilterChange = (value) => {
-    setSelectedFilter(value);
+function ResultsFilter({
+  genresFilterArray,
+  selectedFilters = [],
+  setSelectedFilters,
+  sortingOrder = "desc",
+  sortingType = "vote_average",
+  sortAndFilterResults,
+}) {
+  const handleGenreFilterChange = (item) => {
+    let newSelectedFilters;
+    if (
+      item.filterName === "All" &&
```

Comment:
@Marysosh "All" should be constant because I see more than 3 places where you are using this value

Reply:
Done✅